numpy/core/fromnumeric.py

```diff
@@ -2782,6 +2782,10 @@ def alen(a):
     7
 
     """
+    # NumPy 1.17.0, 2019-08-01
```

Comment:
Should be 1.18.0, we tend not to backport deprecations

Reply:
Oh, sorry. I will change that!